examples/graphbolt/download.py

```diff
@@ -0,0 +1,577 @@
+import argparse, hashlib, os, shutil, tarfile, yaml
+import subprocess
+import urllib.request as ur
+
+import dgl.graphbolt as gb
+import numpy as np
+from tqdm import tqdm
+
+GBFACTOR = 1 << 30
+
+
+def build_yaml_helper(path, dataset_size, in_memory=True):
+    """The stirng to build the yaml file. (Still need modification)"""
+
+    data = {
+        "dataset_name": os.path.basename(path),
+        "feature_data": [
+            {
+                "domain": "node",
+                "format": "numpy",
+                "in_memory": in_memory,
+                "name": "feat",
+                "path": "data/paper_feat.npy",
+            }
+        ],
+        "graph": {
+            "edges": [
+                {"format": "numpy", "path": "edges/paper__cites__paper.npy"}
```

Comment:
Let's add reverse edges to this file following:
https://github.com/mlcommons/training/blob/cdd928d4596c142c15a7d86b2eeadbac718c8da2/graph_neural_network/compress_graph.py#L47

Reply:
For this one, if I add rev_cite, so the there are two edgeds, it will enter the case where it is heterogeneous graph in `on_disk_dataset.py`. Is it what we want?

Reply:
If you check the link above, they do this:
`cites_edge = add_self_loops(remove_self_loops(paper_paper_edges)[0])[0]`
we should do the same.

Reply:
And then they do this: https://github.com/mlcommons/training/blob/cdd928d4596c142c15a7d86b2eeadbac718c8da2/graph_neural_network/compress_graph.py#L49

Reply:
But I think they made a mistake here, they should have concatenated first, then remove self loops and add self loops back.

Reply:
Otherwise, the self loop will be included twice.

Reply:
To clarify, we want only 1 edge type.